apps/forklift/lib/forklift/init.ex

```diff
@@ -2,31 +2,42 @@ defmodule Forklift.Init do
   @moduledoc """
   Task to initialize forklift and start ingesting each previously recorded dataset
   """
-  use Task, restart: :transient
+  use GenServer
 
   alias Forklift.MessageHandler
-  import Forklift
 
+  @name :forklift_init_server
```

Comment:
should this be a function call that extends the application name set by the user (if they set one explicitly)?

Reply:
Maybe? The ability to name it as a user is a testing thing. It should be `:forklift_init_server` for all production releases IMO, which is why it's not a configurable thing via `runtime.exs`.

Reply:
Maybe? The ability to name it as a user is a testing thing. It should be `:forklift_init_server` for all production releases IMO, which is why it's not a configurable thing via `runtime.exs`.

Reply:
Fair enough.